ethcore/res/ethereum/callisto.json

```diff
@@ -0,0 +1,83 @@
+{
+	"name": "Callisto",
+	"dataDir": "Callisto",
+	"engine": {
+		"Ethash": {
+			"params": {
+				"minimumDifficulty": "0x020000",
+				"difficultyBoundDivisor": "0x0800",
+				"durationLimit": "0x0d",
+				"blockReward": "0x16c4abbebea0100000",
+				"homesteadTransition": "0x0",
+				"callistoTransition": "0x0",
```

Comment:
`callistoTransition` does nothing. please remove it from code or clarify what that achieves.